### PR TITLE
LPS-133795

### PR DESCRIPTION
**Pull request by mlan0**
Related Issue: https://issues.liferay.com/browse/LPS-133795

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hey @mlan0, can you please fix the source formatter issues and resend? Thanks

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js [hunk 1]
```diff
@@ -272,7 +272,7 @@ const DatePicker = ({
 						return onChange('');
 					}
 
-					if (moment(value).isValid()) {
+					if (moment(value, getLocaleDateFormat(locale), true).isValid()) {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js:275]: @mlan0, can you please double check if we can remove the parameter `true`?

Comment (mlan0) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js:275]: We can't. I tested and it results in a unexpected behavior.

The moment documentation also says something about it in the following example `moment('2016 is a date', 'YYYY-MM-DD').isValid() //returns true, 2016 was matched
`

**Comment by natocesarrego** [comment]
ci:forward

**Comment by liferay-forms** [comment]
Hey @mlan0, can you please check if these failures are related to your changes? Thanks

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**